manifests/init.pp

```diff
@@ -89,21 +107,31 @@
 #   available, likely results in performance degradation due to I/O blocking and is not recommended in most cases. Modifying this parameter should
 #   be done incrementally with benchmarking at each step to determine an optimal value for your deployment.
 #
-# @example
+# @example Default configuration
 #   include pulpcore
+#
+# @example Disable static content
```

Comment:
@ekohl can you help me understand this use case? why would an user want to disable management of static content ?

Reply:
Static content is only the CSS/JS and you don't need that for the API. Right now in our deployment we don't even serve it and we've shipped that to users.

Reply:
@ekohl My understanding is that it does allow for a nicer presentation of the api docs... Is that something that we don't expect a Katello user to interact with directly?

Reply:
My thinking is that from a troubleshooting standpoint that would be nice to have in the Katello scenario, particularly given that there isn't a `pulp-admin` equivalent for Pulp3.

Reply:
I think so and it is a goal, but right now we don't serve them at all and this allows a lighter deployment.

Reply:
After considering this and seeing that Pulp reads the static files on application startup, I've decided to drop the option to not generate these. I have moved it to its own class so it doesn't restart all applications if anything changes. Thinking about it, it should probably restart the API service if it reads the files.